src/components/Poll/Pair/VoteModal/index.tsx

```diff
@@ -21,34 +21,49 @@ export const VoteModal: React.FC<VoteModalProps> = ({ handeClose, item }) => {
             <h3 className="font-Inter text-2xl font-bold">{item.name}</h3>
             <Close className="cursor-pointer" onClick={handeClose} />
           </header>
-          <div className="self-start rounded-lg border border-black-3 p-1">
-            <div className="flex items-center gap-2 rounded-lg border border-gray-10 px-3 py-1">
-              {item.numOfChildren ? <Layers /> : <Browser />}
-              <span className="font-IBM text-sm">
-                {item.numOfChildren
-                  ? `${item.numOfChildren} + Projects`
-                  : 'Single project'}
-              </span>
+          {item.numOfChildren > 0 && (
+            <div className="self-start rounded-lg border border-black-3 p-1">
+              <div className="flex items-center gap-2 rounded-lg border border-gray-10 px-3 py-1">
+                <Layers />
+                <span className="font-IBM text-sm">
+                  {`${item.numOfChildren} + Projects`}
+                </span>
+              </div>
             </div>
-          </div>
+          )}
         </div>
-        <div className="mb-[73px]  flex max-h-[500px] flex-col overflow-auto">
-          <div className="flex gap-7">
-            <img
-              alt={item.name}
-              className="h-[360px] w-[360px] shrink-0 rounded-2xl"
-              height={360}
-              src={item.image}
-              width={360}
-            />
-            <div className="flex  grow flex-col gap-6 overflow-auto  font-Inter text-lg">
+        <div className="mb-[45px]  flex max-h-[500px] flex-col overflow-auto">
+          <div className="flex justify-between">
+            <div className="flex min-w-[360px] flex-col gap-4">
+              <img
+                alt={item.name}
+                className="w-full shrink-0 rounded-2xl"
+                src={item.image || '/nip.png'}
+              />
+              <div>
+                <span className='font-medium'> Website: </span>
+                <a href={item.url} rel="noreferrer" target='_blank'>{item.url}</a>
+              </div>
+            </div>
+            <div className="flex flex-col gap-6 overflow-auto px-4 font-Inter text-lg">
               <div className="flex flex-col gap-2">
                 <p
                   dangerouslySetInnerHTML={{
-                    __html: item.description || item.name,
+                    __html:
+                      `<b>Impact Description:</b> ${item.impactDescription}` ||
+                      item.name,
                   }}
                 />
               </div>
+              {item.contributionDescription && (
+                <div className="flex flex-col gap-2">
+                  <p
+                    dangerouslySetInnerHTML={{
+                      __html: `<b>Contribution Description:</b> ${item.contributionDescription}`,
```

Comment:
Do you handle line breaks in text? `\n` should be converted to `<br/>`

<img width="972" alt="image" src="https://github.com/GeneralMagicio/pw-frontend/assets/9698363/eb764df2-ec8f-46d8-9c29-e6d310d230ea">